### ISSUE TITLE
Only shows local database

### ISSUE THREAD

**Issue by DarkRedman** [mongo-express/mongo-express#375]
Mongo-Express only shows local database but we can't see the others even using the admin mode. I don't understand why there is such a limitation.

**Comment by dozoisch** [comment]
@DarkRedman I'm not totally sure what you mean. If you change the configuration, you can see any database you want. By default it connects to your localhost, but you can make it connect to anything you want.

**Comment by jlindberg-oss** [comment]
@dozoisch I think this is a recent regression.  I've used mongo-express in the past without issue, but I just downloaded a fresh install (0.44.0) on a new machine and I'm seeing the same behavior as @DarkRedman.

No matter what settings I run with it shows only the database named "local":
![screen shot 2017-11-18 at 7 18 07 am](https://user-images.githubusercontent.com/32934079/32980234-b3f81080-cc30-11e7-958b-e8c8a4edf3e8.png)

Even though the mongod that I'm pointing it at has multiple other databases.
